inference-engine/src/transformations/src/transformations/op_conversions/simplify_ctc_greedy_decoder_seq_len.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "itt.hpp"
+
+#include "transformations/op_conversions/simplify_ctc_greedy_decoder_seq_len.hpp"
+
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::SimplifyCTCGreedyDecoderSeqLen, "SimplifyCTCGreedyDecoder", 0);
+
+ngraph::pass::SimplifyCTCGreedyDecoderSeqLen::SimplifyCTCGreedyDecoderSeqLen() {
+    MATCHER_SCOPE(SimplifyCTCGreedyDecoderSeqLen);
+    auto decoder = pattern::wrap_type<opset6::CTCGreedyDecoderSeqLen>();
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
+        auto decoder_seq_len = std::dynamic_pointer_cast<opset6::CTCGreedyDecoderSeqLen> (m.get_match_root());
+        if (!decoder_seq_len) {
+            return false;
+        }
+        element::Type seq_len_type = decoder_seq_len->input_value(1).get_element_type();
+        // Transposing input data channels from [N, T, C] to [T, N, C]. Need for compatible with CTCGreedyDecoder v1
+        auto transpose = std::make_shared<ngraph::opset6::Transpose>(decoder_seq_len->input_value(0),
+                                                                     ngraph::opset6::Constant::create(element::i32,
+                                                               Shape({3}), {1, 0, 2}));
+        // Receive time and batch dimensions and concatenate to [T, N] tensor shapes:
```

Comment:
nit: `// Receive time and batch dimensions and concatenate to [T, N] tensor shapes:` -> `// Receive time and batch dimensions and concatenate to [T, N] tensor shapes`